src/Process/LintProcess.php

```diff
@@ -86,6 +94,19 @@ public function getSyntaxError()
         return false;
     }
 
+    /**
+     * @return string
+     */
+    public function getSeverity()
+    {
+        if (!empty($this->severity)) {
+            return $this->severity;
+        } else {
+            $error = $this->getSyntaxError();
+            return '$this->severity';
```

Comment:
Why the quotes here?

Reply:
good catch! Fixed in 4f2d28a

Reply:
It would be great to have a unit test to cover this case.